src/client/dfs/cont.c

```diff
@@ -83,7 +84,8 @@ dfs_cont_create(daos_handle_t poh, uuid_t *cuuid, dfs_attr_t *attr, daos_handle_
 			dattr.da_chunk_size = DFS_DEFAULT_CHUNK_SIZE;
 
 		if (attr->da_hints[0] != 0) {
-			strncpy(dattr.da_hints, attr->da_hints, DAOS_CONT_HINT_MAX_LEN);
+			/* DAOS-17042 Replace strncpy with strncat or strlcpy */
```

Comment:
please remove all these comments from the code. they are not helpful

Reply:
Done